src/apos/components/ApostropheTiptapImportData.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="apostrophe-tiptap-file-upload-control">
+    <button @click="click()" class="apos-tiptap-menubar__button" :class="{ 'apos-tiptap-menubar__button--active': active() }">
+      <FileUpload :title="tool.label" :size="18" />
+    </button>
+    <ApostropheModal v-if="editing">
+      <template slot="body">
+        <div class="apos-window-form">
+          <form v-if="editing">
+            <fieldset class="apos-window-fieldset">
+              <label class="apos-window-label" for="file">Upload CSV or Excel file - This will replace previous content in your RichText editor</label>
```

Comment:
"Upload a CSV or Excel file — this will replace all previous content in the rich text editor with a new table"

Reply:
Yes, the example in the README is correct as it was what I used in the demo project